libs/auth/src/artifact_authentication.ts

```diff
@@ -164,35 +165,15 @@ function deserializeArtifactSignatureBundle(
  */
 async function validateSigningMachineCert(
   config: ArtifactAuthenticationConfig,
-  signingMachineCert: Buffer,
-  artifact: ArtifactToImport
-): Promise<void> {
-  const certDetails = await parseCert(signingMachineCert);
-  switch (artifact.type) {
-    case 'cast_vote_records': {
-      assert(
-        certDetails.component === 'central-scan' ||
-          certDetails.component === 'scan',
-        'Signing machine cert for cast vote records should be a VxCentralScan or VxScan cert'
-      );
-      break;
-    }
-    case 'election_package': {
-      assert(
-        certDetails.component === 'admin',
-        'Signing machine cert for election package should be a VxAdmin cert'
-      );
-      break;
-    }
-    /* istanbul ignore next: Compile-time check for completeness */
-    default: {
-      throwIllegalValue(artifact, 'type');
-    }
-  }
```

Comment:
Moved to the `performArtifactSpecificAuthenticationChecks` function below, which I think is a better fit for this logic